components/raftstore/src/store/memory.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use lazy_static::lazy_static;
+use memory_trace_macros::MemoryTraceHelper;
+use std::sync::Arc;
+use tikv_alloc::{mem_trace, trace::MemoryTraceNode};
+
+lazy_static! {
+    pub static ref RAFTSTORE_MEM_TRACE: Arc<MemoryTraceNode> = mem_trace!(
+        raftstore,
+        [(raft_router, [alive, leak]), (apply_router, [alive, leak])]
+    );
```

Comment:
Something like this is possible:
```rust
lazy_static! {
    static ref X: HashMap<usize, usize> = hashmap!{1 => 1};
    static ref Y: usize = *X.get(&1).unwrap();
}
```
So I suggest to define all of `MEMTRACE_ROOT`, `MEMTRACE_RAFT_ROUTER_ALIVE` and `MEMTRACE_RAFT_ROUTER_LEAK` here.